server/echo_handler.cc

```diff
@@ -0,0 +1,22 @@
+#include "echo_handler.h"
+#include "httpRequest.h"
+#include "httpResponse.h"
+#include "http.h"
+#include <string>
+#include <iostream>
+
+RequestHandler::Status EchoHandler::Init(const std::string& uri_prefix,
+				      const NginxConfig& config) {
+  return OK;
+}
+
+RequestHandler::Status EchoHandler::HandleRequest(const Request& request,
+					       Response* response) {
+  std::cout << "A\n";
+  response->SetStatus(Response::OK);
+  std::cout << "A\n";
+  response->AddHeader("Content-Type", http::mime_type::ContentTypeAsString(http::mime_type::CONTENT_TYPE_TEXT_PLAIN));
+  std::cout << "A\n";
```

Comment:
I believe that the input I hardcoded for the input for my unit test was badly formatted (that's my guess). I'm looking into it now but here's the request: 

"GET /echo HTTP/1.1\r\nHost: localhost"